operator/pkg/version/version.go

```diff
@@ -128,6 +138,26 @@ func (v *CompatibilityMapping) UnmarshalYAML(unmarshal func(interface{}) error)
 			return err
 		}
 	}
+
+	if tmp.K8sClientVersionRange != "" {
+		if v.K8sClientVersionRange, err = goversion.NewConstraint(tmp.K8sClientVersionRange); err != nil {
+			return err
+		}
+	} else {
+		if v.K8sClientVersionRange, err = goversion.NewConstraint(">=0"); err != nil {
```

Comment:
seems better to me to leave this as nil rather than filling with ">=0" and just assume there's no constraint if the field is unset when testing.

Reply:
fixed.